projects/kit/components/line-clamp/line-clamp.template.html

```diff
@@ -1,10 +1,9 @@
 <div
-    *tuiLet="lineClamp$ | async as lineClamp"
     tuiHintCustomPosition
     class="t-wrapper"
     [tuiHint]="computedContent"
-    [style.-webkit-line-clamp]="lineClamp"
-    [style.word-break]="(lineClamp || 0) > 1 ? 'break-word' : 'break-all'"
+    [style.-webkit-line-clamp]="lineClamp$ | async"
+    [style.word-break]="((lineClamp$ | async) || 0) > 1 ? 'break-word' : 'break-all'"
```

Comment:
two subscriptions of the same observable :(

Can we avoid it somehow ?

Reply:
Why `*tuiLet` is not appropriate solution ?

Reply:
fixed